### PR TITLE
build.xml cleanup

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Cleanup some of the unused bits in the legacy `build.xml` file.

Targets dependency graphs:

before:

build

after:

build

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: java/manager-build.xml [hunk 1]
```diff
@@ -632,4 +632,16 @@
...
+    <copy file="${rhn-home}/conf/eclipse/.project-template" tofile="${rhn-home}/.project" overwrite="false" />
+    <copy toDir="${rhn-home}" overwrite="false">
+      <fileset dir="${rhn-home}/conf/eclipse/">
+        <include name=".checkstyle" />
+        <include name=".classpath" />
+        <include name=".settings/*" />
+        <!-- include name=".externalToolBuilders/*" /-->
+      </fileset>
+    </copy>
+  </target>
```
Comment (deneb-alpha) [inline java/manager-build.xml:647]: one of the checks complained  too
```suggestion
  </target>
</project>
```

Comment (cbosdo) [reply java/manager-build.xml:647]: doh! I got trick by `git rebase` two times... will be good now

**Comment by mcalmer** [comment]
Some tests are failing. I would suggest to run the acceptance testsuite as it build the package and we see if build.xml works

**Comment by cbosdo** [comment]
> Some tests are failing. I would suggest to run the acceptance testsuite as it build the package and we see if build.xml works

Yes, I was too prompt at removing the ivy properties. The acceptance tests should pass now. And yes I will run the jenkins CI to ensure the package builds

**Comment by cbosdo** [comment]
Now the tests are green and the package builds

**Review by mcalmer — APPROVED** [review]
Let's give it a try

**Pull request merged**